automation/Jenkins/openvino_version.yml

```diff
@@ -0,0 +1 @@
+openvino_wheels_version: 2022.1.0.dev20220316
```

Comment:
let's move to the `automation` because it is not only `Jenkins` specific, but  setup dev environment too

Reply:
This folder contains our other version files and other `yml` configuration files.